src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -1465,6 +1465,8 @@ sein.";
 
 "ContactDiary_Overview_Risk_Text_2" = "Diese müssen nicht in Zusammenhang mit den von Ihnen erfassten Personen und Orten stehen.";
 
+"ContactDiary_Overview_Risk_Text_3" = "aufgrund von mehreren Begegnungen mit niedrigem Risiko";
```

Comment:
Shouldn't there also be a "." at the end? And maybe we could make it clearer, that this is an alternative to text 1? Maybe we could rename them to something like "ContactDiary_Overview_Risk_Text_StandardCause" (Text1), "ContactDiary_Overview_Risk_Text_LowRiskEncountersCause" (Text3) and "ContactDiary_Overview_Risk_Text_Disclaimer" (Text2)?

Reply:
Good catch. Let me change it.